documentation/builders/update.md

```diff
@@ -12,6 +12,98 @@ Restore your old files after the new installation was successful and check if ne
 $ diff shared/settings/jukebox.yaml resources/default-settings/jukebox.default.yaml
 ```
 
+## Manually upgrade to the latest version
+
+> [!CAUTION]
+> For optimal system updates, it is strongly recommended to utilize the upgrade feature when transitioning to the next version. Manual updates may necessitate specific migration steps and, if overlooked, could result in system failure.
```

Comment:
Should we state in which cases this pull method will not be sufficient (e.g. changes to the installation)?
Or should we add this information to the commits details, so it can be looked up?

Reply:
Similar to your comment below, I will state that this method will only work for small updates within the `future3/develop` branch.